src/Microsoft.Security.Utilities.Benchmarks/SecretMaskerDetectionBenchmarks.cs

```diff
@@ -5,42 +5,85 @@
 
 namespace Microsoft.Security.Utilities.Benchmarks
 {
-    public abstract class SecretMaskerDetectionBenchmarks
+    public enum RegexEngine
     {
-        // The size of randomized data to add as a prefix
-        // for every secret. This is intended to make positive
-        // hit less concentrated in the profiling.
-        private const int SecretPrefixSize = 100 * 1024;
+        DotNet,
+        RE2,
+    }
+
+    public enum Case
+    {
+        // This is close to the worst case scenario for large input. Every
```

Comment:
>is [](http://example.com/codeflow?start=16&length=2)

maybe add these as simple <summary> elements? SEV4, why not

Reply:
Ah, `<summary>` was not in backticks and treated as html in your comment. Edited to render correctly, and now I get it, and yes, will do.

Reply:
Good to know this quirk! I am in code flow and it renders everything find, the `<summary>` item displays properly.